### ISSUE TITLE
Accessibility - PDF files not PDF/UA compliant

### ISSUE THREAD

**Issue by tasharghyrou** [LandRegistry/local-land-charges#3]
Currently some of the PDF files found in the Migration Hub are not fully PDF/UA compliant. Some PDF files do not meet the basic requirements criteria due to font not being embedded. There have also been issues reported with tagged content and artifacts. 

There are also some issues with the logical structure of some PDF files such as structure elements missing bounding boxes and alternative descriptions.

Some PDF files have been flagged to have missing PDF/UA identifiers which is classed as a Metadata and Settings issue. All of these issues may affect the order in which the screen reader reads text from the PDF files affected, it may not always be in the logical order.

Please see the PDF's and their PAC reports below, PAC 3 PDF viewer can be downloaded using the following link to generate these reports again (it's in a .zip file and has to be put on a Windows machine) - https://www.access-for-all.ch/en/pdf-accessibility-checker.html

![An Overview for using our Migration Helper](https://user-images.githubusercontent.com/88780180/129324813-823e31d7-
[Business Change Playbook (Make it Ready)_PAC_Report.pdf](https://github.com/LandRegistry/local-land-charges/files/6980834/Business.Change.Playbook.Make.it.Ready._PAC_Report.pdf)
![Business Change Playbook (Make it Stick)](https://user-images.githubusercontent.com/88780180/129324819-ea11802c-a39c-4b1d-8b7e-5f2a92b7a539.JPG)
[Business Change Playbook (Make it Stick)_PAC_Report.pdf](https://github.com/LandRegistry/local-land-charges/files/6980835/Business.Change.Playbook.Make.it.Stick._PAC_Report.pdf)
![Handover Pack](https://user-images.githubusercontent.com/88780180/129324824-ff40d1cd-7a97-4645-b493-c16de34a089d.JPG)
[Handover pack_PAC_Report.pdf](https://github.com/LandRegistry/local-land-charges/files/6980837/Handover.pack_PAC_Report.pdf)
![How business rules can help cleanse your data](https://user-images.githubusercontent.com/88780180/129324826-dc423fc6-4548-4770-815d-84c3d7f018a8.JPG)
[How business rules can help cleanse your data_PAC_Report.pdf](https://github.com/LandRegistry/local-land-charges/files/6980838/How.business.rules.can.help.cleanse.your.data_PAC_Report.pdf)
[How to approach extracting your electronic and digital data _PAC_Report.pdf](https://github.com/LandRegistry/local-land-charges/files/6980839/How.to.approach.extracting.your.electronic.and.digital.data._PAC_Report.pdf)
![How to approach extracting your electronic and digital data](https://user-images.githubusercontent.com/88780180/129324831-94bde287-f342-4c09-bdc8-dfcd3cb71fd0.JPG)
![How to approach Positional Accuracy Issues (PAI) in your data](https://user-images.githubusercontent.com/88780180/129324833-8847ad74-848d-42d8-9b57-54b8a7b84cbb.JPG)
[How to approach Positional Accuracy Issues (PAI) in your data_PAC_Report.pdf](https://github.com/LandRegistry/local-land-charges/files/6980840/How.to.approach.Positional.Accuracy.Issues.PAI.in.your.data_PAC_Report.pdf)
![How to approach scanning your data](https://user-images.githubusercontent.com/88780180/129324835-45b2db7a-daac-4eb3-8e35-b69d79ba2e89.JPG)
[How to approach scanning your data_PAC_Report.pdf](https://github.com/LandRegistry/local-land-charges/files/6980841/How.to.approach.scanning.your.data_PAC_Report.pdf)
![Information to Support your business case](https://user-images.githubusercontent.com/88780180/129324837-cdc2b14a-315c-4a03-b027-f8390082cd90.JPG)
[Information to support your business case_PAC_Report.pdf](https://github.com/LandRegistry/local-land-charges/files/6980842/Information.to.support.your.business.case_PAC_Report.pdf)
![Business Change Playbook (Make it Ready)](https://user-images.githubusercontent.com/88780180/129324817-dc611830-6115-452d-a973-eed6195b61c5.JPG)
a9e5-4a15-a536-14666a9337bc.JPG)
[Business Change Playbook (Make it Happen)_PAC_Report.pdf](https://github.com/LandRegistry/local-land-charges/files/6980833/Business.Change.Playbook.Make.it.Happen._PAC_Report.pdf)
![Business Change Playbook (Make it Happen)](https://user-images.githubusercontent.com/88780180/129324816-75c12b51-6afa-4de1-95cb-0fb83619fec9.JPG)
[An overview for using our Migration Helper _PAC_Report.pdf](https://github.com/LandRegistry/local-land-charges/files/6980831/An.overview.for.using.our.Migration.Helper._PAC_Report.pdf)

**Comment by chrisdebian** [comment]
Looks like this may no longer be actionable — the pages this issue references (the homepage and `migration/index.md`, `key-learnings.md`, `guidance.md`, `news.md`) now just redirect to https://www.gov.uk/government/publications/local-land-charges-migration. The migration hub content described here doesn't appear to be served from this site any more, so there's nothing left to apply heading tags / fix PDF accessibility on.

Flagging in case this is worth closing as resolved-by-decommissioning — happy to be corrected if I'm missing something.
